### PR TITLE
Change references to projectatomic/libpod to containers/libpod

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by vrothberg** [comment]
@rhatdan was removing `vendor/github.com/ulikunitz/xz` intentional? I assume `vndr` removed that as there's no reference on it anymore.

**Comment by vrothberg** [comment]
It looks like we're having a bootstrapping problem as both, buildah and libpod, need to change the references but both vendor code of the other:
```
vendor/github.com/containers/libpod/pkg/secrets/secrets.go:13:2: cannot find package "github.com/projectatomic/libpod/pkg/rootless" in any of:
```

I suggest to run to run the tests locally here and force merge. @rhatdan @TomSweeneyRedHat wdyt?

**Comment by vrothberg** [comment]
I think another (maybe better) solution is to symlink or copy libpod also to `vendor/github.com/projectatomic/libpod` for a short time.

**Comment by TomSweeneyRedHat** [comment]
Maybe it's time to move buildah to containers too?  Not sure if that will help or not, but at least it would all be the same....

**Comment by rhatdan** [comment]
I was hoping to wait til next week.

**Comment by giuseppe** [comment]
I had already a PR for doing the renaming: https://github.com/projectatomic/buildah/pull/927, going to close it

**Comment by rhatdan** [comment]
@TomSweeneyRedHat @nalind @vrothberg @umohnani8 @pixdrift Lets get this merged so we can further fix up libpod.

File: vendor/github.com/docker/docker/client/image_create.go [hunk 1]
```diff
@@ -21,6 +22,9 @@ func (cli *Client) ImageCreate(ctx context.Context, parentReference string, opti
 	query := url.Values{}
 	query.Set("fromImage", reference.FamiliarName(ref))
 	query.Set("tag", getAPITagFromNamedRef(ref))
+	if options.Platform != "" {
+		query.Set("platform", strings.ToLower(options.Platform))
+	}
```
Comment (lsm5) [inline vendor/github.com/docker/docker/client/image_create.go]: is this change related to projectatomic->containers rename?

Comment (lsm5) [reply vendor/github.com/docker/docker/client/image_create.go]: same comment for some other diffs below.

Comment (rhatdan) [reply vendor/github.com/docker/docker/client/image_create.go]: These were vendored in to make the build work.

**Comment by haircommander** [comment]
LGTM

**Comment by umohnani8** [comment]
LGTM

**Comment by rhatdan** [comment]
@rh-atomic-bot r=umohnani8
